src/mount.rs

```diff
@@ -49,6 +49,13 @@ pub fn mount_root(options: &CmdlineOptions) -> Result<()> {
         return Err("root= not found in /proc/cmdline".into());
     }
 
+    if options.rootfstype != Some("nfs".to_string()) &&
+        options.rootfstype != Some("9p".to_string())
```

Comment:
`cargo fmt` complains about that. But maybe there is a better way to do this? 
@hnez do you have any suggestions?

Reply:
(I've posted my reply as a separate comment - I must have clicked the wrong buttons)